ztp/siteconfig-generator/siteConfig/filesHandler.go

```diff
@@ -120,3 +126,14 @@ func GetExtraManifestResourceFiles(manifestsPath string) ([]os.FileInfo, error)
 	}
 	return files, err
 }
+
+func CheckFileName(filePath string) error {
```

Comment:
SiteConfig is deprecated starting in 4.18. I'm fine with the check here but we need to correlate this with behavior in ClusterInstance CR to provide the same protections.